docs/releases/v0.27.0.md

```diff
@@ -0,0 +1,24 @@
+# Changelog
+
+## [v0.27.0](https://github.com/aws-observability/aws-otel-collector/tree/v0.27.0) (2023-03-02)
+
+[Full Changelog](https://github.com/aws-observability/aws-otel-collector/compare/v0.25.1...v0.27.0)
+
+**Closed issues:**
+
+- Support for kubernetes 1.24 and dockershim deprecation [\#1769](https://github.com/aws-observability/aws-otel-collector/issues/1769)
+
+**Merged pull requests:**
+
+- Update all upstream dependencies to 0.72.0 and breaking changes [\#1889](https://github.com/aws-observability/aws-otel-collector/pull/1889) ([humivo](https://github.com/humivo))
+- Include main module in dependabot pr merger [\#1880](https://github.com/aws-observability/aws-otel-collector/pull/1880) ([rapphil](https://github.com/rapphil))
+- Only consider golang for merging dependabot prs [\#1878](https://github.com/aws-observability/aws-otel-collector/pull/1878) ([rapphil](https://github.com/rapphil))
+- Add dependabot PR merger script tool [\#1872](https://github.com/aws-observability/aws-otel-collector/pull/1872) ([rapphil](https://github.com/rapphil))
+- Merging release/v0.26.x to main [\#1870](https://github.com/aws-observability/aws-otel-collector/pull/1870) ([PaurushGarg](https://github.com/PaurushGarg))
+- \[Chore\] - Merge release v0.25.x to main [\#1851](https://github.com/aws-observability/aws-otel-collector/pull/1851) ([vasireddy99](https://github.com/vasireddy99))
+- \[Chore\] - Merge release v0.26.x to main [\#1850](https://github.com/aws-observability/aws-otel-collector/pull/1850) ([vasireddy99](https://github.com/vasireddy99))
+- Fix sed command in bump version script [\#1827](https://github.com/aws-observability/aws-otel-collector/pull/1827) ([bryan-aguilar](https://github.com/bryan-aguilar))
+- \[chore\] merge relase/v0.26.x onto main [\#1815](https://github.com/aws-observability/aws-otel-collector/pull/1815) ([bryan-aguilar](https://github.com/bryan-aguilar))
+- Update Readme links for docs [\#1812](https://github.com/aws-observability/aws-otel-collector/pull/1812) ([vasireddy99](https://github.com/vasireddy99))
```

Comment:
These can be removed. They are not necessary for the release notes.

Reply:
Ok I have fixed it in latest commit